lib/node_modules/@stdlib/stats/base/dists/frechet/logpdf/src/main.c

```diff
@@ -0,0 +1,56 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/stats/base/dists/frechet/logpdf.h"
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/ln.h"
+#include "stdlib/math/base/special/pow.h"
+#include "stdlib/constants/float64/ninf.h"
+#include "stdlib/constants/float64/nan.h"
+
+/**
+* Returns the logpdf for a Fréchet distribution with shape `alpha`, scale `s`, and location `m` at a value `x`.
+*
+* @param x    	  x parameter
+* @param alpha    shape parameter
+* @param s        scale parameter
+* @param m        location parameter
+* @return         logpdf
```

Comment:
```suggestion
* @return         evaluated logPDF
```

Let's follow the same comment structure as in `main.js`